packages/app/src/systems/Core/components/PublicRoute/PublicRoute.tsx

```diff
@@ -9,20 +8,14 @@ type PublicRouteProps = {
 };
 
 export function PublicRoute({ children }: PublicRouteProps) {
-  const { accounts, isLoading } = useAccounts();
+  const { accounts } = useAccounts();
   const location = useLocation();
   const match = useResolvedPath(location.pathname);
   const isSignUp = match.pathname.includes("/sign-up");
 
-  if (isLoading) {
-    return (
-      <BoxCentered minWS minHS>
-        <Spinner />
-      </BoxCentered>
-    );
-  }
-  if (!isSignUp && !isLoading && accounts?.length) {
+  if (!isSignUp && accounts != null && accounts?.length) {
```

Comment:
Why remove the loading state? It's in another place now?

Reply:
It was triggering re-renders when you refetch account info

Home page is taking care of loading